src/app/sharing/[id]/page.tsx

```diff
@@ -0,0 +1,54 @@
+import Image from 'next/image';
+import Link from 'next/link';
+import { notFound } from 'next/navigation';
+import React from 'react';
+
+import { AcceptButton } from './AcceptButton';
+
+import { TopNav } from '@/components/TopNav';
+import { BackIcon } from '@/components/icons/BackIcon';
+import { executeSharing, fetchSharing } from '@/services/shareService';
+
+export default async function Page({ params }: { params: { id: string } }) {
+    const { sharing } = await fetchSharing(params.id);
+    if (!sharing) {
+        notFound();
+    }
+
+    const backBtn = (
+        <Link href={`/`}>
+            <BackIcon />
+        </Link>
+    );
+
+    const handleAccept = async () => {
+        'use server';
+        return await executeSharing(params.id);
+    };
+
+    return (
+        <div className="w-96 min-h-screen p-4 grid grid-rows-[auto_1fr] gap-10">
+            <TopNav left={backBtn} />
+            <div className="w-full justify-self-center flex flex-col items-center gap-5">
+                <div className="w-10/12 flex flex-col items-center gap-1">
```

Comment:
This div on line 33 is not needed. 
The div under TopNav (line32) can contain 3 blocks: Title, Card image and Button. `w-full` (line32) should be changed to `w-10/12` while the `w-10/12` for Button (line48) should be changed to `w-full`. Also, you will not need the `mt-4`.

Reply:
Fixed: https://github.com/tsoibet/kaado/pull/25/commits/eac871a681175792a084b5172c847a387f29b69f